docs/modules/reference/pages/daemons/daemon-config-files/provisiond.adoc

```diff
@@ -75,17 +75,23 @@ NOTE: Any changes to the thread pools require a restart of the {page-component-t
 This pool determines the number of requisitions that can be imported at one time.
 Increase this number if you have a large number of requisitions.
 
+Import threads will check the requisition against the database to determine what add/change/delete operations are necessary.
```

Comment:
The Import threads will compare the Nodes in the requisition with the Nodes in the database determine what Node Insert, Update, or Delete operations are necessary. The delete operations are scheduled first, followed by the updates, and finally the inserts. 

(Using the same terminology as in the code. Not wedded to that thinking, though)

Reply:
Import threads are for the requisitions. The synch button in the UI or the import schedule in provision-configuration.xml